### PR TITLE
Use DateTimePicker widget from XDSoft which also supports time picking.

### PR DESCRIPTION
**Pull request by mbaechtold**
See our internal tracker for more information: https://extranet.4teamwork.ch/dev/4teamwork-egov/tracker-4teamwork-egov/845

### CODE REVIEW

**Comment by maethu** [comment]
Just one last question, what about multi lingual sites? Or event sites, where it's possible to change the UI language?

Since the jquery widgets supports this feature, we should implement this on the backend.
So we can use different formats for different languages.

First check if it's possible to have a ``format`` configuration on every language. 
Than get the language from the language cookie if available (JS and Serverside)

File: ftw/datepicker/profiles/default/jsregistry.xml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0"?>
+<object name="portal_javascripts" meta_type="JavaScripts Registry" autogroup="False">
```
Comment (phgross) [inline ftw/datepicker/profiles/default/jsregistry.xml]: For the profile changes an upgradestep is needed.

Comment (mbaechtold) [inline ftw/datepicker/profiles/default/jsregistry.xml]: Thank you, @phgross. I've added an upgrade step.

**Comment by phgross** [comment]
Date (only) fields aren't handled correctly (tested with `opengever.core`).

bildschirmfoto 2015-08-14 um 08 53 06

**Comment by phgross** [comment]
Die Monate sind noch nicht übersetzt.

**Pull request merged**